package.json

```diff
@@ -41,7 +41,8 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
+    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --only_4xx --ignore-status-codes 403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
+    "proof:all": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' ./_site",
```

Comment:
> [!note]
> I created this `proof:all` script to allow us to manually scan for all status codes. We might find that we want to continue to refine the `--ignore-status-codes` as time goes on, but I am considering that a follow-on conversation for this PR.

Reply:
LGTM! I tried adding `--log_level` flags and only saw a difference when using `debug` and `fatal`.

Debug listed the status code of _every_ link which was interesting, but `fatal` simply provided the final error count without flagging each failed link

```js 
// added `--log_level=fatal` to script

npm run proof:all

> proof:all
> bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0 --swap-urls 'https\://designsystem.digital.gov/:/' --ignore-files '/whats-new/updates/(2017|2018|2019|2020|2021)/,/documentation/code-guidelines/' --log_level=fatal ./_site


HTML-Proofer found 25 failures!
```

I'm assuming this just makes the log less verbose and would still count as a failure.